classwrap.d.ts

```diff
@@ -0,0 +1,17 @@
+export as namespace classwrap
+
+export interface ClassSet {
+  [key: string]: boolean | any | ClassSet
+}
+
+export type Class = string | number | ClassSet
```

Comment:
Here I defined what are available type for a class. To make it able to use it alone or in an array type.